src/bp/ui-studio/src/web/components/Layout/index.tsx

```diff
@@ -131,25 +136,42 @@ class Layout extends React.Component<ILayoutProps> {
       'go-module-nlu-entities': () => this.gotoUrl('/modules/nlu/Entities')
     }
 
+    const splitPanelLastSizeKey = `BP/UI/${window.BOT_ID}/BOTTOM_PANEL_SIZE`
```

Comment:
I think we should follow the standard with keys, actually there is bp::botId::thekey for bot-scoped variables

Reply:
ok will change